syntax = "proto3";

package neuron.neurotransmitter;
option go_package = "neuron/interop";

message SerializedStuff {
  bytes content = 1;
}

message AsymmetricKey {
  SerializedStuff secret = 1;
  SerializedStuff public_key = 2;
}

